tests/test_scripts.py

```diff
@@ -54,6 +55,21 @@ def test_load_into_counting():
     assert os.path.exists(outfile)
 
 
+def test_load_into_counting_nonwritable():
+    script = scriptpath('load-into-counting.py')
+    args = ['-x', '1e3', '-N', '2', '-k', '20', '-t']
+
+    outfile = utils.get_test_data('test-nonwritable')
+    os.chmod(outfile, stat.S_IWOTH | stat.S_IRUSR)
```

Comment:
Hm. Alright. I'll do that then.

On Sat, Apr 4, 2015, 17:08 Michael R. Crusoe <notifications@github.com>
wrote:

> In tests/test_scripts.py
> <https://github.com/ged-lab/khmer/pull/892#discussion_r27772145>:
>
> > @@ -54,6 +55,21 @@ def test_load_into_counting():
> >      assert os.path.exists(outfile)
> >
> >
> > +def test_load_into_counting_nonwritable():
> > +    script = scriptpath('load-into-counting.py')
> > +    args = ['-x', '1e3', '-N', '2', '-k', '20', '-t']
> > +
> > +    outfile = utils.get_test_data('test-nonwritable')
> > +    os.chmod(outfile, stat.S_IWOTH | stat.S_IRUSR)
>
> (env)mcrusoe@athyra:~/khmer/gl-master$ touch tests/test-data/test-nonwritable
> (env)mcrusoe@athyra:~/khmer/gl-master$ chmod a-w tests/test-data/test-nonwritable
> (env)mcrusoe@athyra:~/khmer/gl-master$ git add tests/test-data/test-nonwritable
> (env)mcrusoe@athyra:~/khmer/gl-master$ git commit -m 'test'
> [counting-test 54da2de] test
>  1 file changed, 0 insertions(+), 0 deletions(-)
>  create mode 100644 tests/test-data/test-nonwritable
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/ged-lab/khmer/pull/892/files#r27772145>.
>